praw/models/reddit/live.py

```diff
@@ -37,3 +60,73 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
 
     def _info_path(self):
         return API_PATH['liveabout'].format(id=self.id)
+
+    def updates(self, **generator_kwargs):
+        """Return a :class:`.ListingGenerator` yields :class:`.LiveUpdate` s.
+
+        :param generator_kwargs: keyword arguments passed to
+            :class:`.ListingGenerator` constructor.
+        :returns: A :class:`.ListingGenerator` object which yields
+            :class:`.LiveUpdate` object.
+        """
+        url = API_PATH['live_updates'].format(id=self.id)
+        for update in ListingGenerator(self._reddit, url,
+                                       **generator_kwargs):
+            update._thread = self
```

Comment:
Yes, definitely fine. All comments for a submission share the same Submission instance.